docs/readthedocs/source/doc/LLM/Quickstart/webui_quickstart.md

```diff
@@ -123,6 +123,41 @@ Enter prompts into the textbox at the bottom and press the **Generate** button t
 To shut down the WebUI server, use **Ctrl+C** in the **Anaconda Prompt** terminal where the WebUI Server is runing, then close your browser tab.
 
 
+## 5. Advanced Usage
+### Using Instruct Chat Mode
```

Comment:
is `Instruct Chat` confusing? since there is a mode option `chat-instruct`?
Just `Chat Mode`?

Reply:
It should be `instruct` mode. Maybe change to `Using Instruct mode for Chat`?

Reply:
Yeah, I think instruct mode is fine